packages/clerk-js/src/ui/signUp/signUpFormHelpers.ts

```diff
@@ -194,7 +194,7 @@ function getPhoneNumberField({
 }: FieldDeterminationProps): Field | undefined {
   if (isProgressiveSignUp) {
     // If there is no ticket and phone number is enabled, we have to show it in the SignUp form
-    const show = !hasTicket && attributes.phone_number.enabled;
+    const show = attributes.phone_number.enabled;
```

Comment:
Leaving a note to my future self, this fix has been transferred to the v4 component